test-package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "ffc-ahwr-payment",
+  "version": "1.0.0",
+  "description": "FFC Vet Visits Payment Service",
+  "homepage": "https://github.com/DEFRA/ffc-ahwr-payment",
+  "main": "app/index.js",
+  "type": "commonjs",
```

Comment:
To revisit in New year perhaps - I don;t know why the package.json with module in it will not execute the tests, but this works around it for now

Reply:
fixed